README.md

```diff
@@ -2700,14 +2716,16 @@ Other Style Guides
         .call(tron.led);
 
     // good
-    const leds = stage.selectAll('.led')
-        .data(data)
-      .enter().append('svg:svg')
-        .classed('led', true)
-        .attr('width', (radius + margin) * 2)
+    const leds = stage
+      .selectAll('.led')
+      .data(data)
+      .enter()
+      .append('svg:svg')
+      .classed('led', true)
+      .attr('width', (radius + margin) * 2)
       .append('svg:g')
-        .attr('transform', `translate(${radius + margin}, ${radius + margin})`)
-        .call(tron.led);
+      .attr('transform', `translate(${radius + margin}, ${radius + margin})`)
+      .call(tron.led);
```

Comment:
All right. Maybe add a Note with recommendation when to add additional indentation before chained methods calls? 
Because I am sure question about such formatting is asked by many who read those "good" examples.

Reply:
I restore previous version. If you have thought about possible note here please let me know